Tests/Unit/Models/RemoteCategoryTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ShopwarePlugins\Connect\Tests\Unit\Components;
+
+use Shopware\CustomModels\Connect\RemoteCategory;
+use Shopware\CustomModels\Connect\RemoteToLocalCategory;
+use Shopware\Models\Category\Category;
+
+class RemoteCategoryTest extends \PHPUnit_Framework_TestCase
```

Comment:
Do we need dedicated test for entity ? I think we can just make assertions where we test ```CategoryResolver::storeRemoteCategories```

Reply:
we make that tests 
i added them because test coverage did go down